docs/docs/features/dongle.mdx

```diff
@@ -0,0 +1,349 @@
+
+    config ZMK_KEYBOARD_NAME
+        default "My Board"
+
+    config ZMK_SPLIT_ROLE_CENTRAL
+        default y
+
+    config ZMK_SPLIT_BLE_CENTRAL_PERIPHERALS
+        default 2
+
+    # Prevent the dongle from going to sleep
+    config ZMK_SLEEP
+        default n
+
+    # Increase the transmit power of the dongle
+    config BT_CLTR_TX_PWR_PLUS_8
+        default y
+
+endif
+
+
+if SHIELD_MY_BOARD_LEFT
+
+    # Setting this to y will make the left half the central
+    # We want it to be the central when dongle is not used
+    config ZMK_SPLIT_ROLE_CENTRAL
+        default y
+
+    # Check if the left half is the central
+    if ZMK_SPLIT_ROLE_CENTRAL
+        config ZMK_KEYBOARD_NAME
+            default "My Board"
+            
+    endif # ZMK_SPLIT_ROLE_CENTRAL
+
+endif # SHIELD_MY_BOARD_LEFT
+
+if SHIELD_MY_BOARD_DONGLE || SHIELD_MY_BOARD_LEFT || SHIELD_MY_BOARD_RIGHT
+
+    config ZMK_SPLIT
+        default y
+
+endif
+```
+
+</TabItem>
+<TabItem value="unibody">
+
+Now we will add the dongle configuration to the unibody keyboard.
+We will default the unibody keyboard to not use the dongle configuration.
+
+```kconfig title="Kconfig.defconfig"
+if SHIELD_MY_BOARD_DONGLE
+
+    config ZMK_KEYBOARD_NAME
+        default "My Board"
+
+    config ZMK_SPLIT_ROLE_CENTRAL
+        default y
+
+    config ZMK_SPLIT
+        default y
+
+    # Prevent the dongle from going to sleep
+    config ZMK_SLEEP
+        default n
+
+    # Increase the transmit power of the dongle
+    config BT_CLTR_TX_PWR_PLUS_8
+        default y
+
+endif
+
+if SHIELD_MY_BOARD
+    config ZMK_KEYBOARD_NAME
+        default "My Board"
+    
+    config ZMK_SPLIT
+        default n
+    
+endif
+```
+
+</TabItem>
+</Tabs>
+
+### Dongle Overlay
+
+:::note
+Dongle overlay configuration is same for both split and unibody keyboards.
+:::
+
+In most common cases the dongle will not have any keys, in that case we can instead use a mock kscan module to simulate the keys.
+
+Here we will define a mock kscan module with no keys and a default matrix transform.
+The matrix map needs to match the keyboard matrix map.
+
+```dts title="my_board_dongle.overlay"
+#include <dt-bindings/zmk/matrix_transform.h>
+
+/ {
+    chosen {
+        zmk,kscan = &mock_kscan;
+        zmk,matrix-transform = &default_transform;
+    };
+
+    mock_kscan: kscan_0 {
+        compatible = "zmk,kscan-mock";
+        columns = <0>;
+        rows = <0>;
+        events = <0>;
+    };
+
+    default_transform: keymap_transform_0 {
+        compatible = "zmk,matrix-transform";
+        columns = <3>;
+        rows = <3>;
+        map = <
+                RC(0, 0) RC(0, 1) RC(3, 0)
+                RC(1, 0) RC(1, 1) RC(1, 2)
+                RC(2, 0) RC(2, 1) RC(2, 2)
+        >;
+    };
+};
+```
+
+:::note
+Without the matrix transform, the dongle will not be able to communicate with the keyboard.
+You will get an error message similar to `zmk: No behavior assigned to 7 on layer 0`.
+:::
```

Comment:
```suggestion
```
Unnecessary warning imo, assuming the matrix transform part is written out better like in my previous suggestion.